machine_test.go

```diff
@@ -250,6 +256,10 @@ func TestJailerMicroVMExecution(t *testing.T) {
 	}
 
 	m.StopVMM()
+
+	info, err := os.Stat(fifoWriterPath)
```

Comment:
Why are you stating the writer here?

Reply:
just to ensure that the fifo was actually created. Just want to ensure that that logic is correct.

Reply:
You're creating the fifo in the test (not the code under test) on line 150.  Why do you then need to assert that it's created?

(Stat doesn't tell you anything about the contents, either.)

Reply:
Not created, just that something has been written to it

Reply:
Okay, the name confused me.  This isn't a fifo, but the destination file where the data passed through a fifo ends up being written (in `captureFifoToFile`).  Can you change the name of this to reflect what it actually is?  Something like capturedLog?

Reply:
Done!